src/allotropy/parsers/thermo_fisher_nanodrop_eight/nanodrop_eight_parser.py

```diff
@@ -253,10 +266,31 @@ def _get_measurement_document(
                     ),
                 )
             )
+
         a280_col = "A280"
         if a280_col not in data.columns and "A280 10mm" in data.columns:
             a280_col = "A280 10mm"
+
         if _get_str_or_none(data, row, a280_col):
+            # capture concentration on the A280 measurement document if the experiment type is
+            # something other than DNA or RNA
+            mass_concentration = None
+            if na_type is not None and "NA" not in na_type and concentration_col:
+                mass_concentration = _get_concentration(
+                    _get_float(data, row, str(concentration_col)),
+                    _get_str_or_none(data, row, "Units"),
+                )
+            processed_data_aggregate_document = None
+            if mass_concentration:
+                processed_data_aggregate_document = ProcessedDataAggregateDocument(
+                    processed_data_document=[
+                        ProcessedDataDocumentItem(
+                            # capture concentration on the A280 measurement document if the experiment type is
+                            # something other than DNA or RNA
+                            mass_concentration=mass_concentration
+                        )
+                    ]
+                )
```

Comment:
Out of scope for this PR, but could you please extract some helpers for this method? `_get_measurement_document` is now over 100 lines long. It was hard to tell, for instance, that these declarations
```
      na_type = _get_str_or_none(data, row, "NA Type")
      concentration_col = self._get_concentration_col(data)
```
needed to be outside the first `if` block.

Reply:
@bdworth I was tempted to make those changes but then I realized I'd end up rewriting a good part of this parser. This PR is supposed to only make changes to the schema.

There is actually an issue with the way we are parsing the `mass_concentration` that is outside the scope of this PR and I plan to handle after discussing with Joe.